index.html

```diff
@@ -323,6 +323,253 @@ <h2 class="h2">opacity</h2>
+      <div class="flex flex-col shadow-md rounded-md">
+        <div class="overlay">
+          <div class="badge-container img-container">
+            <img
+              class="rounded-md img-responsive"
+              src="https://picsum.photos/350/250"
+              alt=""
+              srcset="https://picsum.photos/350/250"
+            />
+            <span
+              class="absolute font-bold card-badge badge bg-grey-500 text-white"
+              >Out of Stock</span
+            >
+            <button class="btn-close bg-rose-500"></button>
+          </div>
+          <div class="flex flex-col p-5">
+            <div class="flex justify-between align-items-center">
+              <h5 class="h5 font-semibold">Title</h5>
+              <i class="fas fa-heart text-grey-500 text-hover-rose-500"></i>
+            </div>
+            <div class="flex mt-3 mb-10">
+              <p class="subtitle1 line-through mr-3">Price</p>
+              <p class="subtitle1 text-rose-500">price</p>
+            </div>
+
+            <a
+              class="btn btn-disabled bg-rose-600 bg-hover-rose-400 w-36 text-white"
+              ><span><i class="btn-icon fas fa-shopping-cart"></i></span>Add to
+              Cart</a
+            >
+          </div>
+        </div>
+      </div>
+    </div> -->
+
+    <!-- inputs -->
+    <!-- <div class="test-2">
+      <div class="input-container">
+        <input
+          type="text"
+          id="email"
+          class="input-outlined input-outlined-red"
+          autocomplete="off"
+          placeholder=" "
+        />
+        <label for="email" class="input-label bg-grey-100">Email</label>
+      </div>
+    </div>
+    <div class="test-2">
+      <div class="input-container">
+        <input
+          type="text"
+          id="email"
+          class="input-outlined input-outlined-pink"
+          autocomplete="off"
+          placeholder=" "
+        />
+        <label for="email" class="input-label bg-grey-100">Email</label>
+      </div>
+    </div>
+    <div class="test-2">
+      <div class="input-container">
+        <input
+          type="text"
+          id="email"
+          class="input-outlined input-outlined-blue"
+          autocomplete="off"
+          placeholder=" "
+        />
+        <label for="email" class="input-label bg-grey-100">Email</label>
+      </div>
+    </div>
+    <div class="test-2">
+      <form action="">
+        <div  class="input-container">
+          <input
+            type="text"
+            id="email"
+            required
+            class="input-outlined text-lg text-black input-outlined-rose"
+            autocomplete="off"
+            placeholder=" "
+          ></input>
+
+          <label for="email" class="input-label text-grey-700 bg-grey-100"
+            >Email *
+          </label>
+
+          <small class="error text-red-500 text-sm">Error Message</small>
+        </div>
+      </form>
+    </div> -->
+    <div class="test-2">
```

Comment:
Suggestions:
Do add a comment here in order to track out where the Lists related component details are!!

Reply:
yes, will do that